style.css

```diff
@@ -1,62 +1,226 @@
-body {
-  background: url('aust copy.jpg');
-  background-size: cover;
-  /* position: fixed; */
+/* JS Navbar Scrolling CSS */
+.navbar.scrolled {
+	background: #a1cbdf;
+	transition: background 400ms;
 }
-
-.splash {
-  font-family: 'Noto Sans', sans-serif;
-  padding-top: 60px;
+#city {
+	background: url('building.png');
```

Comment:
Try adding `center` here.
```css
background: url('somefile.jpg') center;
```

It'll make a difference when you shrink to mobile width, that is, if you want that building to be in the center of the screen.

<img width="947" alt="Screen Shot 2021-01-24 at 23 26 13" src="https://user-images.githubusercontent.com/7801456/105665022-af5d4800-5e9b-11eb-9a3d-80b016842e52.png">

Reply:
(I added center in my dev tools)